.travis.yml

```diff
@@ -8,21 +8,13 @@ php:
     - 7.2
     - nightly
 
+dist: trusty
+
 # run build against nightly but allow them to fail
 matrix:
     fast_finish: true
     allow_failures:
         - php: nightly
-    include:
-        - php: 5.3
-          dist: precise
-          sudo: required
-        - php: 5.4
-          dist: precise
-          sudo: required
```

Comment:
5.3 and 5.4 version are supported then please do not remove it.

Reply:
I don't have much experience with Travis but I think the previous configuration wasn't testing 5.3 and 5.4 already?

Reply:
That's possible but let's run tests on all supported php version but at least on the first one.

Reply:
Hum, not https://travis-ci.org/FriendsOfSymfony1/symfony1/builds/577819519

Look at the end of the list.

Reply:
You are right. As I said, I don't have much experience with Travis.

The issue seems to be that 5.3&5.4 needs to run with precise, but they are not picking up that configuration from .travis.yml. Any idea why?

Reply:
It is executed you can revert https://github.com/FriendsOfSymfony1/symfony1/pull/216/commits/f6a62b7e7c02c36a2499102df20a8824359b775a

And all will going good.